blade.migrate.liferay70/src/blade/migrate/core/XMLFileChecker.java

```
@@ -0,0 +1,91 @@
+package blade.migrate.core;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.Locator;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+/**
+ *
+ * @author andy wu
+ *
+ */
+public class XMLFileChecker extends DefaultHandler {
+
+	private File file;
+	//is in the target Tag
+	private boolean inState = false;
+	private List<SearchResult> results = null;
+	private String tagName;
+	private String value;
+	private Locator locator;
+	private SAXParser parser;
+
+	/**
+	 *
+	 * @param file
+	 */
+	public XMLFileChecker(File file){
+		this.file = file;
+		SAXParserFactory factory = SAXParserFactory.newInstance();
+		try {
+			parser = factory.newSAXParser();
+		} catch (ParserConfigurationException e) {
```

Catch all exceptions and then throw a new IllegalArgumentException(e)